test/disjointCoarseMallocPool.cpp

```diff
@@ -759,3 +759,95 @@ TEST_P(CoarseWithMemoryStrategyTest, disjointCoarseMallocPool_wrong_params_4) {
 
     umfMemoryProviderDestroy(malloc_memory_provider);
 }
+
+TEST_P(CoarseWithMemoryStrategyTest, disjointCoarseMallocPool_split_merge) {
```

Comment:
would be nice to have some negative tests as well

Reply:
@lukaszstolarczuk ping

Reply:
perfect, thank you!